apps/molgenis-components/src/components/forms/RowEdit.vue

```diff
@@ -316,9 +316,14 @@ function executeExpression(expression, values, tableMetaData) {
     }
   });
 
+  const parsedExpression = expression.replaceAll(
```

Comment:
why are you making changes here? This change should be limited to the backend only?

Reply:
> Okay, I undid the 'static' changes while keeping the IdGenerator interface (although I also don't think that is needed because we will have only one autoid impl to start with and maybe even until end of times)

why this makes the code less tightly coupled and hard to maintain